client/components/Editor/plugins/keymap.ts

```diff
@@ -20,12 +20,29 @@ import { wrapInList, splitListItem, liftListItem, sinkListItem } from 'prosemirr
 import { undo, redo } from 'prosemirror-history';
 import { undoInputRule } from 'prosemirror-inputrules';
 import { keymap } from 'prosemirror-keymap';
+import { Selection } from 'prosemirror-state';
 import { mathBackspaceCmd } from '@benrbray/prosemirror-math';
 
 import { splitBlockPreservingAttrs } from '../commands';
 
 const mac = typeof navigator !== 'undefined' ? /Mac/.test(navigator.platform) : false;
 
+const arrowHandler = (dir) => (state, dispatch, view) => {
```

Comment:
Can we get some types on this bad boy?

Reply:
Resolved, I hope, by latest commit. Sometimes I fear over-typing, but I reckon that's misplaced fear.